policyengine_canada/variables/gov/cra/tax/income/credits/canada_employment_amount.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_canada.model_api import *
+
+
+class canada_employment_amount(Variable):
+    value_type = float
+    entity = Person
+    label = "Canada Employment Amount"
+    unit = CAD
+    definition_period = YEAR
+    reference = "https://www.canada.ca/en/revenue-agency/services/tax/individuals/topics/about-your-tax-return/tax-return/completing-a-tax-return/deductions-credits-expenses/line-31260-canada-employment-amount.html"
+
+    def formula(person, period, parameters):
+        max_amount = parameters(
```

Comment:
and remove `.max_amount`
```suggestion
        p = parameters(
```